hs_discover/public/index.html

```diff
@@ -20,10 +20,11 @@
     }
 
     .vdpComponent.vdpWithInput > input {
-        /*width: 233px;*/
+         width: 233px;
```

Comment:
Is this addressing #3997 ?  My css isn't great.

Reply:
Correct. I picked up on a theme in your comments. Going forward in Pull Requests, I will try to remember to tag/mark the bugfix link at the exact commit or lines of code.